pkg/registry/storage/rest/storage_storage.go

```diff
@@ -144,6 +144,15 @@ func (p RESTStorageProvider) v1Storage(apiResourceConfigSource serverstorage.API
 		storage["csinodes"] = csiNodeStorage.CSINode
 	}
 
+	// register csidrivers if CSIDriverRegistry feature gate is enabled
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
```

Comment:
The feature is enabled and can't be disabled, no need to check for it.

Reply:
I've removed these checks wherever they were found. I noticed that there are still some checks for other features (such as CSINodeInfo), should we create an issue to remove these checks as well?